src/modals/lambdaStatisticsModal.js

```diff
@@ -34,9 +35,21 @@ const lambdaStatisticsModal = (screen, blessed, application, lambdaName, cloudwa
         border: 'line',
         style: { fg: 'green', border: { fg: 'green' } },
     });
-    const durationChart = new DurationBarChart(application, cloudwatchLogs)
-    durationChartBox.append(durationChart.chart);
-    durationChart.updateBarChart(lambdaName);
+    const errorChartBox = blessed.box({
+        parent: lambdaStatisticsLayout,
+        width: 55,
+        height: 22,
+        left: 'right',
+        top: 'center',
+        align: 'center',
+        padding: { right: 2, left: 2 },
+        border: 'line',
+        style: { fg: 'green', border: { fg: 'green' } },
+    });
```

Comment:
Could create a chartBox component to use for errorChartBox and durationChartBox?

Reply:
good one, I agree